extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/VertxConfiguration.java

```diff
@@ -33,8 +32,8 @@
      * The maximum amount of time the event loop can be blocked.
      * Default is 2s.
      */
-    @ConfigItem
-    public Optional<Duration> maxEventLoopExecuteTime;
+    @ConfigItem(defaultValue = "2")
```

Comment:
The nice thing is that you can drop the default from the javadoc now. That's why I asked :).